### DISCUSSION TITLE
How to make this kind of tab?

### DISCUSSION THREAD

**Discussion by SnowTechnical** [requarks/wiki#6450] [Guides]
Good morning,
I'm new to wiki.js and I would like to know how to do this tab genge that leads to a link, I couldn't find the info in the guide
Thanks in advance

![wiki](https://github.com/requarks/wiki/assets/99869476/67badd20-f6f8-4ad9-bfde-2bd0af38cd74)

**Comment by fnafox** [comment]
```
# Getting Started

Wiki.js is quick and easy to install. You should be up and running in no time.

- [Requirements *Server and database prerequisites.*](/install/requirements)
- [Installation Guide *Detailed installation instructions for all platforms.*](/install)
{.links-list}
```

Hint, if you click on page actions in the top right corner of the page and click view source, you'll see the plain markdown.

**Reply by SnowTechnical** [reply]
Thank you
